### DISCUSSION TITLE
Error: Failed to delete project: 401

### DISCUSSION THREAD

**Discussion by pcxnet** [AntlerVC/firetable#958] [Support / Q&A] [answered]
👉 **Please describe the steps that you took that led to this bug.**
Can't delete project.   Have emailed support.

**Comment by BeeBombshell** [comment, marked as answer]
Hi @pcxnet. It's a little challenging to see how this isn't working out for you. It'd be great if you can set up a call with @shamsmosowi, so that he can help you figure this out. Here's a link to set up a support call: https://cal.com/shams/rowy

**Comment by pcxnet** [comment]
I've booked one in for Monday
